### DISCUSSION TITLE
How to create a pore network with a specified size distribution?

### DISCUSSION THREAD

**Discussion by mourendong** [PMEAL/OpenPNM#2209] [Q&A] [answered]
Hi, everyone,

I try to create a pore network with a specified size distribution, such as logarithmic normal distribution.
And, in my simulation, the pores or nodes do not have any volume and only serve as a computational grid, and in fact, the throat follows the specified size distribution.

Can you give me some tips on how to create it?

**Comment by ma-sadeghi** [comment, marked as answer]
You need to create a custom geometry and then add a specific model for `throat.diameter`. See [this](https://github.com/PMEAL/OpenPNM/blob/dev/examples/notebooks/geometry/adjusting_pore_size_distributions.ipynb) example for reference. Let me know if you need further help.
